.github/workflows/update-master-after-main.yml

```
@@ -1,4 +1,4 @@
-name: Updates the master branch after main
+name: Updates the main branch after main
```

```suggestion
name: Updates the main branch after master
```
or else i'm confused as to what this workflow will do now

---

actually... it's supposed to remain in the original state: the workflow updates the master branch after changes are pushed to main